game.c

```diff
@@ -304,11 +302,12 @@ void level_step_chemistry(level* lvl) {
                 }
                 inv = inv->next;
             }
-            if (lvl->tiles[x][y] != WALL && lvl->tiles[x][y] != CLOSED_DOOR && lvl->chemistry[x][y]->elements[air] < 20) lvl->chemistry[x][y]->elements[air] += 3;
+            //TODO make constants? tiles regen 3 air if they are below 20?
```

Comment:
Those these be pulled out as constants? If so, I would put these at top of this file for easier tweaking until they are finalized.

Reply:
I suspect it would make sense to centralize a lot of configuration about the behavior of the world, this could be included in that.